bridge-stp.conf

```diff
@@ -1,7 +1,7 @@
 # If 'y' or commented out, mstpd will be automatically started/stopped as
 # needed.  If 'n', mstpd will not be automatically started/stopped by the
 # bridge-stp script.
-MANAGE_MSTPD='n'
+#MANAGE_MSTPD='n'
```

Comment:
squash this commit in  commit 9ac71b1a6f4a4d3f416d5f8a308aa9355c4cd141 ?
it looks a bit redundant to me to add it there, and disable it here